src/server/v2.0/harbor/handler/base/base.go

```diff
@@ -0,0 +1,30 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package base
+
+import (
+	"context"
+
+	"github.com/go-openapi/runtime/middleware"
+)
+
+// API base API handler
+type API struct{}
+
+// SendError returns response for the err
+func (*API) SendError(ctx context.Context, err error) middleware.Responder {
```

Comment:
What is the difference of this method with `api.ServeError = serverError`

Reply:
serveError here used to transform the error format of go-swagger to our common error format.